qiskit/schemas/deprecated/qobj/qobj_generic_schema.json

```diff
@@ -26,7 +26,7 @@
             "title": "fixed 1Q op",
             "required": ["qubits"],
             "properties": {
-                "name": {"enum": ["reset", "id", "h", "s", "sdg", "t", "tdg", "x", "y", "z"]},
+                "name": {"enum": ["reset", "id", "h", "s", "sinv", "t", "tinv", "x", "y", "z"]},
```

Comment:
Aer will need to be updated to expect the new gate names (and report them in `basis_gates`). Can you open an issue there?

Reply:
Qiskit/qiskit-aer#533